### PR TITLE
Add image displaying and filtering

### PR DESCRIPTION
**Pull request by Corramel**

### CODE REVIEW

**Comment by iAmGb-20** [comment]
LGTM! Love the code organization here!

**Review by iAmGb-20 — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -112,7 +112,7 @@
         <artifactId>appengine-maven-plugin</artifactId>
         <version>2.2.0</version>
         <configuration>
-          <projectId>${googleCloudProjectId}</projectId>
+          <projectId>spring21-sps-2</projectId>
```
Comment (matty377) [inline pom.xml:115]: I think this variable is already set on line 19. This doesn't change anything really as long as we don't change the one on line 19 and they go out of sync.

File: src/main/webapp/placeholder.html [hunk 2]
```diff
@@ -6,15 +6,16 @@
 	<link rel="stylesheet" href="style.css">
 	<script src="script.js"></script>
 </head>
-<body>
+<body onload=loadElements()>
```
Comment (matty377) [inline src/main/webapp/placeholder.html:9]: This should have quotes around it, right? Does this work?

File: src/main/webapp/script.js [hunk 3]
```diff
@@ -12,17 +12,125 @@
...
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function appendText(txt, parent) {
+    let p = document.createElement("p");
+    p.innerText = txt;
+    parent.appendChild(p);
+}
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+/* Maybe turn this into (displayAll?) or (display5Random)?
```
Comment (matty377) [inline src/main/webapp/script.js:46]: Do we need to keep the commented out code around?

**Review by matty377 — APPROVED** [review]

**Pull request merged**